### ISSUE TITLE
Issue with PHPUnit-bridge in dev (6.4.x-dev 15607a3) 

### ISSUE THREAD

**Issue by heiglandreas** [symfony/symfony#57071]
### Symfony version(s) affected

6.4.x

### Description

When I run PHPUnit in a symfony project using phpunit-bridge  version 6.4.x-dev 15607a3 I run into this message:

```
PHPUnit 10.5.20 by Sebastian Bergmann and contributors.

Error in bootstrap script: PHPUnit\Event\Code\NoTestCaseObjectOnCallStackException:
Cannot find TestCase object on call stack
#0 [path to]/vendor/phpunit/phpunit/src/Runner/ErrorHandler.php(64): PHPUnit\Event\Code\TestMethodBuilder::fromCallStack()
#1 [path to]/vendor/symfony/phpunit-bridge/DeprecationErrorHandler.php(381): PHPUnit\Runner\ErrorHandler->__invoke(2, 'include([path to]...', '[path to]...', 9)
#2 [path to]/vendor/symfony/phpunit-bridge/DeprecationErrorHandler.php(132): Symfony\Bridge\PhpUnit\DeprecationErrorHandler::Symfony\Bridge\PhpUnit\{closure}(2, 'include([path to]...', '[path to]...', 9, Array)
#3 [path to]/config/bootstrap.php(9): Symfony\Bridge\PhpUnit\DeprecationErrorHandler->handleError(2, 'include([path to]...', '[path to]...', 9)
#4 [path to]/config/bootstrap.php(9): include('[path to]')
#5 [path to]/tests/bootstrap.php(3): require('[path to]')
#6 [path to]/vendor/phpunit/phpunit/src/TextUI/Application.php(313): include_once('[path to]')
#7 [path to]/vendor/phpunit/phpunit/src/TextUI/Application.php(104): PHPUnit\TextUI\Application->loadBootstrapScript('[path to]')
#8 [path to]/vendor/phpunit/phpunit/phpunit(104): PHPUnit\TextUI\Application->run(Array)
#9 [path to]/bin/phpunit(122): include('[path to]')
#10 {main}
```

When I run the same tests but with phpunit-bridge 6.4.7 everything works as expected.

### How to reproduce

Set up a symfony project using phpunit-bridge and run phpunit.

I am still investigating this in more detail

**Comment by xabbuh** [comment]
@HypeMC Related to #54797?

**Comment by heiglandreas** [comment]
It seems to have been introduced with [b21398fd7b69e6a6d25e9b35b2cd201ec89124c4](https://github.com/symfony/phpunit-bridge/commit/b21398fd7b69e6a6d25e9b35b2cd201ec89124c4) 

That is the commit it starts failing with...

**Comment by HypeMC** [comment]
@heiglandreas Could you create a minimum reproducer?

**Comment by heiglandreas** [comment]
Still working on that. First had to fix our setup 😁

**Comment by heiglandreas** [comment]
I was able to create a minimum reproducible case.
Find the repository (including a readme how to reproduce the case) at https://github.com/heiglandreas/symfony-reproducer-57071

The problem seems to be https://github.com/heiglandreas/symfony-reproducer-57071/blob/5bc0eb4c2c6188bda446476f205ee2b3df47d066/tests/bootstrap.php#L7 which calls an `@include` on a file that doesn't exist. That should not be an issue as the error is suppressed via the `@` but seems to now break.

When I create the file in question the error is gone but I can not necessarily assure that the file is available - which is why we used the `@` here...

**Comment by HypeMC** [comment]
@heiglandreas Could you pleas see if #57110 fixes your issue.

**Comment by heiglandreas** [comment]
Can confirm that #57110 fixes the issue!

**Comment by shyim** [comment]
The bug has been released yesterday, could we get the fix merged and released?

**Comment by gnutix** [comment]
I upgraded to 6.4.8 and got this issue as well.

**Comment by SpartakusMd** [comment]
@heiglandreas I have the same issue. Tested the fix and confirm I can run the tests successfully.

**Comment by alebedev80** [comment]
> I upgraded to 6.4.8 and got this issue as well.

me too

**Comment by SpartakusMd** [comment]
@alebedev80 The fix was merged and should be released in 6.4.9.

**Comment by philippeso** [comment]
> @alebedev80 The fix was merged and should be released in 6.4.9.

This will also be released in 5.4.41? Fix of release 5.4.40? (issue detected in paratests too: https://github.com/facile-it/paraunit/issues/248)
